pkg/storaged/dialog.jsx

```diff
@@ -318,6 +319,16 @@ function flatten(arr1) {
     return arr1.reduce((acc, val) => Array.isArray(val) ? acc.concat(flatten(val)) : acc.concat(val), []);
 }
 
+const HelperTextWarning = ({ text }) =>
```

Comment:
Any reason why this does not use the [HelperText](https://www.patternfly.org/v4/components/helper-text/) React component? That would also avoid the manual CSS import.

Reply:
Oh, I somehow thought it doesn't exist...